website/docs/docs/cloud/manage-access/about-access.md

```diff
@@ -8,142 +8,203 @@ pagination_prev: null
+
+  <Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/access-control/post-login-screen.png" width="60%" title="The screen users see after their first SSO login." />
+
+4. They now open their email and click the link to join dbt Labs, which completes the process.
+
+  <Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/access-control/sample-email.png" width="60%" title="The email the user receives on first SSO login." />
+
+Euclid is now logged in to their account. They only have access to the `Jaffle Shop` pr, and the project selection option is removed from their UI entirely. 
+
+<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/access-control/rbac-account-home.png" width="60%" title="The home screen with access restricted." />
+
+They can now configure development credentials. The `Production` environment is visible, but it is `read-only`, and they have full access in the `Staging` environment. 
+
+<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/access-control/production-restricted.png" width="60%" title="The Production environment landing page with read-only access." />
+
+<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/access-control/staging-access.png" width="60%" title="The Staging environment landing page with full access." />
+
+</Expandable>
+
+With RBAC configured, you now have granular control over user access to features across dbt Cloud.
 
 ## FAQs
 
-- **When are IdP group memberships updated for SSO Mapped groups?**  <br />
- Group memberships are updated whenever a user logs into dbt Cloud via a supported SSO provider. If you've changed group memberships in your identity provider or dbt Cloud, ask your users to log back into dbt Cloud to synchronize these group memberships.
-- **Can I set up SSO without RBAC?**  <br />
+<Expandable alt_header="When are IdP group memberships updated for SSO Mapped groups?">
+
+Group memberships are updated whenever a user logs into dbt Cloud via a supported SSO provider. If you've changed group memberships in your identity provider or dbt Cloud, ask your users to log back into dbt Cloud to synchronize these group memberships.
+
+</Expandable>
+
+<Expandable alt_header="Can I set up SSO without RBAC?">
+
 Yes, see the documentation on [Manual Assignment](#manual-assignment) above for more information on using SSO without RBAC.
-- **Can I configure a user's License Type based on IdP Attributes?** <br />
-  Yes, see the docs on [managing license types](/docs/cloud/manage-access/seats-and-users#managing-license-types) for more information.
 
-- **Why can't I edit a user's group membership?**  <br />
-Make sure you're not trying to edit your own user as this isn't allowed for security reasons. To edit the group membership of your own user, you'll need a different user to make those changes.
+</Expandable>
+
+<Expandable alt_header="Can I configure a user's License Type based on IdP Attributes?">
```

Comment:
I think we can use lower case here because it's not a product name. I believe the IdP abbreviations would stay the same. 
```suggestion
<Expandable alt_header="Can I configure a user's license type based on IdP attributes?">
```